ykrt/src/mt.rs

```diff
@@ -219,215 +217,206 @@ impl MTThread {
         &mut self,
         loc: &Location<I>,
     ) -> Option<fn(&mut I) -> *mut RawStopgapInterpreter> {
-        // Since we don't hold an explicit lock, updating a Location is tricky: we might read a
-        // Location, work out what we'd like to update it to, and try updating it, only to find
-        // that another thread interrupted us part way through. We therefore use compare_and_swap
-        // to update values, allowing us to detect if we were interrupted. If we were interrupted,
-        // we simply retry the whole operation.
-
-        // We need Acquire ordering, as PHASE_COMPILING and PHASE_COMPILED need to read information
+        // XXX We need Acquire ordering, as PHASE_COMPILING and PHASE_COMPILED need to read information
```

Comment:
Is the XXX left in accidentally?

Reply:
Fixed in 80db78f86a108897087f83b958a6ede52671685e.